app/src/main/res/values/strings.xml

```diff
@@ -551,7 +552,9 @@
   <string name="bloodsugarhistory_add_blood_sugar_button">Diabetes</string>
   <string name="bloodsugarhistory_blood_sugar_date_time" translatable="false">%1$s\n%2$s</string>
   <string name="bloodsugarhistory_toolbar_title">%1$s, %2$s, %3$s</string>
-  <string name="bloodsugarhistory_blood_sugar_reading" translatable="false">%1$s mg/dl %2$s</string>
+  <string name="bloodsugarhistory_blood_sugar_reading" translatable="false">%1$s %2$s %3$s</string>
+  <string name="bloodsugarhistory_unit_type_mg_dl" translatable="false">mg/dl</string>
```

Comment:
Please remove this as `translatable=false`

Reply:
Transifex is translating the entire mg/dl. That's why I had to mark this is as translatable="false"

Example:
Hindi translation for mg/dl
<string name="bloodsugarentry_md_dl">मिग्रा/डीएल</string>